near-contract-standards/src/non_fungible_token/approval/approval_impl.rs

```diff
@@ -59,7 +61,7 @@ impl NonFungibleTokenApproval for NonFungibleToken {
         // if given `msg`, schedule call to `nft_on_approve` and return it. Else, return None.
         msg.map(|msg| {
             ext_nft_approval_receiver::ext(account_id)
-                .with_static_gas(env::prepaid_gas() - GAS_FOR_NFT_APPROVE)
+                .with_static_gas(Gas(env::prepaid_gas().0.saturating_sub(GAS_FOR_NFT_APPROVE.0)))
```

Comment:
I think you missed a similar place here: https://github.com/near/near-sdk-rs/blob/master/near-contract-standards/src/non_fungible_token/core/core_impl.rs#L399

Reply:
No, that one is checked because of the require above it